### DISCUSSION TITLE
Redirect to a different Domain after Successful Identity Provider Sign-in

### DISCUSSION THREAD

**Discussion by abuelgasimsaadeldin** [keycloak/keycloak#8733] [Q&A]
I want users after signing in with their social logins to be redirected to my apps registration page (custom URL) is that possible?

**Comment by joneswilliam1** [comment]
I would like to accomplish this same exact task, after successful Google Sign in to be re-directed to my Apps Registration page where I verify their phone number (through otp) and request for other user details then sign them in without requiring to set up any password.

But I only want them to go through that registration once and the next time they sign-in using their Google account they will be re-directed to the app straight. Any help on how I could accomplish this would be highly appreciated.  🙏🙏🙏

P.S. my app has already been integrated with Keycloak and upon registration, I store Phone Number as the unique value in 'username', email Address, Name and various other attributes.

Best,
Jones
